doc/man/8/ceph.rst

```diff
@@ -248,6 +248,15 @@ Usage::
 	ceph df {detail}
 
 
+features
```

Comment:
makes sense! but we organize the commands using underlined subsections, and use double underline for the man pages' standard sections, like "Synopsis", "Options", etc . let me think about how to group these two commands in a more consistent way.